### ISSUE TITLE
Possibility to create a cluster without worker nodes

### ISSUE THREAD

**Issue by akoenig** [xetys/hetzner-kube#273]
First of all: Thanks for the great tool! It helps a loooot! :slightly_smiling_face: 

I have one question though: What do you think of adding an option to `cluster create` in which I can create a cluster without any assigned worker nodes? In my use case, I want to add the workers later.

**Comment by xetys** [comment]
Did you try to set the count of worker nodes to 0 by

``` bash
$ hetzner-kube cluster create -n my-cluster ... -w 0
```

?

**Comment by akoenig** [comment]
Yes, I did. It states that at least one worker node has to be defined.
